### PR TITLE
fixed sha256 for coconubattery under macos sierra

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

It will tell you why Travis is failing.

**Comment by julienlavergne** [comment]
Removed unused metadata

**Comment by julienlavergne** [comment]
Fixed sha256

File: Casks/coconutbattery.rb [hunk 1]
```diff
@@ -9,10 +9,8 @@
     url "https://www.coconut-flavour.com/downloads/coconutBattery_#{version}.zip"
   else
     version '3.4'
-    sha256 'cc64cec27cde9677c1313920bea46e05049a2354c0a7ab1b04f18895c80c12fe'
+    sha256 '3b16e5dc3a26a9b3f698743ef891abbe9bba16aa8f540d437f9a7081957e8102'
     url "https://www.coconut-flavour.com/downloads/coconutBattery_#{version.dots_to_underscores}.zip"
-    appcast 'http://updates.coconut-flavour.com/coconutBatteryIntel.xml',
```
Comment (vitorgalvao) [inline Casks/coconutbattery.rb]: Don’t remove this, this is not unused. It is very much useful.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Thank you for the report. Closing in favour of https://github.com/caskroom/homebrew-cask/pull/25349.

Next time, please fill the template, as it included needed checks.

**Pull request closed without merging**